packages/labs/nextjs/README.md

```diff
@@ -0,0 +1,37 @@
+# @lit-labs/ssr-react
+
+A package for integrating Lit SSR with Next.js.
+
+## Overview
+
+Lit components, by default, can be imported and added to Next.js projects but will only be shallowly rendered on the server.
+
+This package provides a plugin for Next.js that incorporates tools from [`@lit-labs/ssr-react`](../ssr-react/README.md) into the project for deep server rendering of Lit components.
+
+## Usage
+
+```js
+// next.config.js
+const withLitSSR = require('@lit-labs/nextjs')();
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // Add your own config here
+  reactStrictMode: true,
+  swcMinify: true,
+};
+
+module.exports = withLitSSR(nextConfig);
+```
+
+## Considerations
+
+The plugin has been tested with Next.js versions 12 and 13. It currently does not support usage with the experimental `app` directory introduced with version 13.
```

Comment:
Is there an issue filed for this? Maybe file and mention here?

Reply:
Done